sentry-ruby/lib/sentry/client.rb

```diff
@@ -199,7 +199,17 @@ def event_from_log(message, level:, **options)
 
       return unless body
 
-      LogEvent.new(level: level, body: body, attributes: attributes, origin: origin)
+      sanitized_attributes = attributes.transform_values do |value|
+        if value.is_a?(String)
+          return unless (sanitized_string = Utils::EncodingHelper.safe_utf_8_string(value))
```

Comment:
@sl0thentr0py WDYT about replacing malformed values with something like `<malformed utf string>` or something like this instead of silently dropping? Feels like a reasonable thing to do. Same re dropping if the body is malformed.